### ISSUE TITLE
Boolean config settings for enabled area types don't work

### ISSUE THREAD

**Issue by jonom** [sheadawson/silverstripe-blocks#30]
ReadMe shows an example of disabling an area type with boolean value but `BlockManager::getAreasForPageType()` only filters on 'only' and 'except' values.

```yml
BlockManager:
  themes:
    simple:
      areas:
        Sidebar: false # Does not work
        Sidebar:
          only: ImaginaryPageType # Works
```

**Comment by sheadawson** [comment]
Interesting... there wasn't any intention for people to use false, and no reason to use false, but I guess the fact it's a boolean value implies you could if you wanted to. I'll address this when I have more time by checking for false value in getAreasForPageType or you could send a PR :)

**Comment by jonom** [comment]
I think there's a good reason to allow false - say if none of your templates include a Sidebar, or if you only have one Blocks area in your templates, you'll want to completely exclude most of the options. Also a bit short on time but if I find some spare before you do I'll send a PR :P

**Comment by sheadawson** [comment]
Hang on though why declare Sidebar: false when you can just not put Sidebar in your area config to begin with?

**Comment by jonom** [comment]
Is that how it's supposed to work? Maybe I'm doing something wrong, but if I do this:

```yml
BlockManager:
  themes:
    default:
      areas:
        AfterContent: true
      use_blocksets: false # Whether to use BlockSet functionality (default if undeclared: true)
      exclude_from_page_types: # Disable the Blocks tab completely on these pages of these types 
        - HomePage 
  use_default_blocks: true # Disable/enable the default Block types (ContentBlock) (default if undeclared: true)
```

I get this:

![screen shot 2015-08-06 at 1 37 01 pm](https://cloud.githubusercontent.com/assets/1079425/9122807/408973a0-3c40-11e5-9782-78cb143c6a8b.png)

So it seems like you have to disable any Areas you don't want to use? By the way my apologies, the ReadMe doesn't show an example of using a value of `false` to disable an Area, I must have been hallucinating!

**Comment by jonom** [comment]
Ergh my brain's not working properly - you're right!

**Comment by sheadawson** [comment]
Hehe all good man :)

On Fri, Aug 7, 2015 at 8:43 AM, Jono Menz <notifications@github.com> wrote:

> Closed #30 <https://github.com/sheadawson/silverstripe-blocks/issues/30>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sheadawson/silverstripe-blocks/issues/30#event-375731328>
> .
>



-- 
*Shea Dawson*
Web Developer
LiveSource

*mobile* +64 20 4028 5647
*skype* squatchnz
*email* shea@livesource.co.nz
*web *www.livesource.co.nz
